src/cpp/database/entities.hpp

```diff
@@ -39,6 +39,15 @@ struct DataReader;
 struct DataWriter;
 struct Locator;
 
+/**
+ * @brief Check whether a specific topic corresponds to a metatraffic topic.
+ *
+ * @param topic_name Name of the topic to check.
+ * @return true if metatraffic topic, false otherwise.
+ */
+bool is_metatraffic_topic(
```

Comment:
Set this function as an `Entity` or `Topic` `static` method

Reply:
:+1: